### DISCUSSION TITLE
training dataset format for lung CT lung nodule detection training

### DISCUSSION THREAD

**Discussion by Rajesh-ParaxialTech** [Project-MONAI/MONAI#7521] [General]
Hello
Thanks for the nice lung CT lung nodule detection training model.
Following is the dataset_fold0.json  file 
![image](https://github.com/Project-MONAI/MONAI/assets/159233894/3b9c94c1-e32b-4505-9b50-52a3fc2bcb98)

under the LUNA16_datasplit  folder

Could you please direct me  as to how the training data format must be ?

My training data is organized as "nii.gz" extension files as follows:

![image](https://github.com/Project-MONAI/MONAI/assets/159233894/55016fa1-48c6-4ba0-bd9a-f28749cfed05)

Thanks

Rajesh

**Comment by KumoLiu** [comment]
Hi @Rajesh-ParaxialTech, for additional information, you may refer to this [section](https://github.com/Project-MONAI/tutorials/tree/main/detection#1-data) in our tutorial. Moreover, you can download the associated JSON files for the LUNA16 dataset directly from this [link](https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/LUNA16_datasplit-20220615T233840Z-001.zip).

Thanks.

**Reply by Rajesh-ParaxialTech** [reply]
Thanks a lot KumoLiu
